util.go

```diff
@@ -9,3 +16,170 @@ func StringPtr(s string) *string {
 func BoolPtr(b bool) *bool {
 	return &b
 }
+
+// containsString returns true if the given list contains the given element.
+func containsString(strings []string, targetString string) bool {
```

Comment:
suggestion: it may be good to clarify what contains string in the function name. `listContainsString` maybe?